generators/typescript-v2/dynamic-snippets/src/EndpointSnippetGenerator.ts

```diff
@@ -1,6 +1,11 @@
 import { FernIr } from "@fern-api/dynamic-ir-sdk";
 import { ts } from "@fern-api/typescript-ast";
 
+const SNIPPET_PACKAGE_NAME = "example";
+const SNIPPET_IMPORT_PATH = "fern";
+const SNIPPET_FUNC_NAME = "do";
+const CLIENT_VAR_NAME = "client";
```

Comment:
These values might not be relevant to TypeScript snippets - they're largely Go-specific. The goal we're after is something like the following:

```typescript
import { CohereClientV2 } from 'cohere-ai';

const cohere = new CohereClientV2({
  token: '<YOUR_API_KEY>',
});
await cohere.chat({
  model: 'command-r-plus',
  messages: [
    {
      role: 'user',
      content: 'hello world!',
    },
  ],
});
```

Reply:
Gotcha, so no need to hold onto them if not required for generating snippets